### PR TITLE
provide jts 1.15 upgrade instructions

### PR DESCRIPTION
**Pull request by jodygarnett**
Introduced headings to seperate out different GeoTools 20 upgrade activities as was done for prior releases with more than one API change

### CODE REVIEW

File: docs/user/welcome/upgrade.rst [hunk 1]
```diff
@@ -29,139 +29,167 @@ But first to upgrade - change your dependency to |release| (or an appropriate st
...
+
+The transitive dependency will correctly bring in the required jars::
+
+     <dependency>
+        <groupId>org.locationtech.jts</groupId>
+        <artifactId>jts-core</artifactId>
+        <version>${jts.version}</version>
+     </dependency>
+
+**Package change to org.locationtech.org**
```
Comment (jnh5y) [inline docs/user/welcome/upgrade.rst]: org.locationtech.org should be org.locationtech.jts

File: docs/user/welcome/upgrade.rst [hunk 2]
```diff
@@ -29,139 +29,167 @@ But first to upgrade - change your dependency to |release| (or an appropriate st
...
+  
+  This is a checked exception, in areas of the GeoTools library where this was found we now return an IllegalArgument exception.
   
-* Getting Unit instances
+* Search ``converter == UnitConverter.IDENTITY`` replace ``converter.isIdentity()``
+* Search ``javax.measure.unit.NonSI`` replace ``import si.uom.NonSI``
+* Search ``javax.measure.unit.SI`` replace ``import si.uom.SI``
+* Search ``SI.METER`` replace ``SI.METRE``
+* Search ``javax.measure.converter.UnitConverter`` replace ``javax.measure.UnitConverter``
+* Search ``javax.measure.unit.UnitFormat`` replace ``import javax.measure.format.UnitFormat``
```
Comment (jnh5y) [inline docs/user/welcome/upgrade.rst]: Does JSR-363 have both si.uom.* and javax.measure.* packages?

**Comment by jnh5y** [comment]
Looks good to me.  I asked @elahrvivaz to review it as well, and it makes sense to him as well.

File: docs/user/welcome/upgrade.rst (hunk 2 above)
Comment (jodygarnett) [reply docs/user/welcome/upgrade.rst]: jar-363 has javax.measure.* packages.

You need to choose your own implementation - we chose si.uom.*

**Comment by aaime** [comment]
The JTS part looks good, I'm confused by the UOM one, seems like it's removing some examples and the like but could be reformatting. As a rule of thumb we suggest to keep large reformats in separate commits. At least some description of what changed there would be useful (the commit message and PR description only talk about JTS).

**Comment by jodygarnett** [comment]
The reformatting was done to break out subheadings for JTS instructions and UOM would not trip on each other

**Pull request merged**